front-end/src/components/Chart.js

```diff
@@ -1,18 +1,20 @@
 import '../styles/chart.css'
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import Highcharts from 'highcharts'
 import HighchartsReact from 'highcharts-react-official'
 import Boost from 'highcharts/modules/boost';
-//import Papa from "papaparse";
+import HighchartsColorAxis from "highcharts/modules/coloraxis";
+require('highcharts-multicolor-series')(Highcharts);
```

Comment:
Im sure this is not the right format but Im not sure what the alternative is to match it